docs/LOCALIZATION.md

```diff
@@ -0,0 +1,251 @@
+# Localization
+
+The components in the boilerplate are fully localized for all languages supported by Enterprise Aurora. As such, all property labels, dropdown options, placeholders, etc. show up in the editor in the same language as the rest of the application. This chapter describes how localization in the component set has been implemented and how new custom components can leverage that same mechanism.
+
+## Basics
+
+Localization is based on the use of string keys instead of hardcoded values for properties in `components-definition.json` and on the use of interpolations in HTML templates. Translations for the keys are provided per language in the `localization` folder within the component set.
+
+## Supported languages
+
+Overview of the languages supported by Enterprise Aurora and their matching translation file names.
+
+| Language | Filename |
+| ------------- | ------------- |
+| Czech  | `csCZ.json` |
+| German  | `deDE.json` |
+| English  | `enUS.json` |
+| Spanish  | `esES.json` |
+| Finnish  | `fiFI.json` |
+| French  | `frFR.json` |
+| Italian  | `jaJP.json` |
+| Korean  | `koKR.json` |
+| Dutch  | `nlNL.json` |
+| Polish  | `plPL.json` |
+| Portuguese  | `ptBR.json` |
+| Russian  | `ruRU.json` |
+| Simplified Chinese  | `zhCN.json` |
+| Traditional Chinese  | `zhTW.json` |
+
+## Localization of `components-definition.json`
+
+### Component with a hardcoded UI string
+
+The label property simply contains a string which will be shown directly.
+
+```javascript
```

Comment:
javascript -> json?

Reply:
Github shows the comments in the json as errors. Not so nice.